src/components/date-range-picker/date-range-picker.spec.ts

```diff
@@ -0,0 +1,102 @@
+import { Component } from '@angular/core';
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { FormsModule } from '@angular/forms';
+import { PopoverModule } from '../../components/popover';
+import { IconModule } from '../icon';
+import { DateRangePickerModule } from './date-range-picker.module';
+
+
+
+@Component({
+    selector: 'ux-components-date-range-picker',
+    template: `
+        <ux-date-range-picker
+            [(start)]="start"
+            [(end)]="end"
+            [showTime]="showTime"
+            [showTimezone]="showTimezone"
+            [showSeconds]="showSeconds"
+            [showMeridian]="showMeridian"
+            [showSpinners]="showSpinners"
+            [showNowBtn]="showNowBtn"
+            [startTimezone]="startTimezone"
+            [endTimezone]="endTimezone"
+            (startChange)="onRangeChange()"
+            (endChange)="onRangeChange()"
+            (keydown.escape)="popover.hide(); input.focus()"
```

Comment:
Had another look at these karma tests, `startChange` and `endChange` outputs are both calling the same function possibly it would be better to call them `startChange` and `endChange` so there is no confusion?